src/work-allocation-2/containers/task-list/task-list.component.ts

```diff
@@ -184,8 +186,14 @@ export class TaskListComponent implements OnChanges {
   }
 
   public onResetSorting(): void {
+    let header = document.getElementById(`header_${this.sortedBy.fieldName}`);
+    header.setAttribute('aria-sort', 'none');
+    this.pagination.page_number = 1;
     this.setDefaultSort();
     this.sortEvent.emit(this.sortedBy.fieldName);
+    this.paginationEvent.emit(this.pagination.page_number);
+    header = document.getElementById(`header_${this.sortedBy.fieldName}`);
+    header.setAttribute('aria-sort', 'ascending');
   }
```

Comment:
This method is interesting... Why do you set the aria-sort attribute and then immediately take them off?

Reply:
Have added comments